nextflow_schema.json

```diff
@@ -119,7 +129,205 @@
+                },
+                "differential_feature_name_column": {
+                    "type": "string",
+                    "default": "gene_name",
+                    "description": "Where a features file (GTF) has been provided, what attributed to use to name features"
+                },
+                "differential_foldchanges_logged": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Indicate whether or not fold changes are on the log scale (default is to assume they are)"
+                }
+            },
+            "required": [
+                "differential_file_suffix",
+                "differential_feature_id_column",
+                "differential_fc_column",
+                "differential_qval_column",
+                "differential_min_fold_change",
+                "differential_max_qval"
+            ],
+            "fa_icon": "fas fa-adjust"
+        },
+        "deseq2_specific_options_rna_seq_only": {
+            "title": "DESeq2 specific options (RNA-seq only)",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "deseq2_test": {
+                    "type": "string",
+                    "default": "Wald",
+                    "description": "'test' param passed to DESeq()",
+                    "help_text": "either \"Wald\" or \"LRT\", which will then use either Wald significance tests (defined by nbinomWaldTest), or the likelihood ratio test on the difference in deviance between a full and reduced model formula (defined by nbinomLRT)"
+                },
+                "deseq2_fit_type": {
+                    "type": "string",
+                    "default": "parametric",
+                    "description": "'fitType' param passed to DESeq()",
+                    "help_text": "either \"parametric\", \"local\", \"mean\", or \"glmGamPoi\" for the type of fitting of dispersions to the mean intensity. See estimateDispersions for description."
+                },
+                "deseq2_sf_type": {
+                    "type": "string",
+                    "default": "ratio",
+                    "description": "'sfType' param passed to DESeq()",
+                    "help_text": "either \"ratio\", \"poscounts\", or \"iterate\" for the type of size factor estimation. See estimateSizeFactors for description."
+                },
+                "deseq2_min_replicates_for_replace": {
+                    "type": "integer",
+                    "default": 7,
+                    "help_text": "the minimum number of replicates required in order to use replaceOutliers on a sample. If there are samples with so many replicates, the model will be refit after these replacing outliers, flagged by Cook's distance. Set to Inf in order to never replace outliers. It set to Inf for fitType=\"glmGamPoi\".",
```

Comment:
'It set to Inf for fitType=\"glmGamPoi\"."' Do you mean 'Is set to Inf', i.e. if the fitType is glmGamPoi, this param is automatically set to inf?

Reply:
This is a direct lift from the DESeq2 docs for which I take no responsibility ;-)